tests.clj

```diff
@@ -154,5 +154,10 @@
         (is (= (slurp (fs/file "test-resources/new/my-scratch/deps.edn"))
                (slurp (fs/file (str target-dir "/deps.edn")))))))))
 
+(deftest clj-neil-new-test
+  (let [{:keys [out err]} (sh "clojure -M:neil new --help" {:dir "tests-clj"})]
```

Comment:
I think we could just use `babashka.deps/clojure` here. This also makes it work on Windows.